### DISCUSSION TITLE
Ubuntu RustDesk Upgrade

### DISCUSSION THREAD

**Discussion by risaw1981** [rustdesk/rustdesk#8684] [Q&A] [answered]
Hey all

Im a complete Linux novice, was trying to upgrade my RustDesk last night with the command line provided in the guides but I kept seeing the permission failed even though its accepting the sudo password. Any ideas would be appreciated! Is it my user no being a sudo user?

**Comment by Nawaki89** [comment, marked as answer]
have you tried to login with **sudo -i** this does that the terminal is logged in with root privileges..  

to upgrade Rustdesk PRO server

sudo -i 
< your password >
bash <(wget -qO- https://raw.githubusercontent.com/rustdesk/rustdesk-server-pro/main/update.sh)

**Reply by risaw1981** [reply]
Thanks, I’ll give that a try this weekend! I’ve stumbled across something similar before. Strange how the account I installed Ubuntu with isn’t a sudo user. I’ll update you on the results. 

Thanks again!

**Reply by risaw1981** [reply]
Thanks for your help sudo -i worked

**Comment by xlionjuan** [comment]
I'm just using this:

```
cd /tmp && wget https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.7-x86_64.deb && sudo dpkg -i rustdesk-1.2.7-x86_64.deb
```

**Comment by risaw1981** [comment]
Sorry should have been more specific, Im upgrading the self hosted pro server, Im assuming doing this will disconnect any current connections so havent been able to run the task yet. I didnt want to schedule it as I want to monitor it in case of issues.
